<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tybh_sub</class>
 <widget class="QWidget" name="Tybh_sub">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>新增项目</string>
   </property>
  </widget>
  <widget class="closeButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>0</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>721</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>新增项目图样信息</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>项目代号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_proNum">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>项目负责人：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>产品代号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_productNum">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>产品名称：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_productName">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>项目名称：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>368</x>
      <y>160</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>项目类别：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_proName">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_proLeader">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_proType">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton
{
    color:white;
    background-color:rgb(14 , 150 , 254);
    border: 1px solid rgb(11 , 137 , 234);
}

QPushButton#pushButton:hover
{
    color:white;
    background-color:rgb(44 , 137 , 255);
    border: 1px solid rgb(11 , 137 , 234);
}

QPushButton#pushButton:pressed
{
    color:white;
    background-color:rgb(14 , 135 , 228);
    border: 1px solid rgb(12 , 138 , 235);
    padding-left:3px;
    padding-top:3px;
}
</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>closeButton</class>
   <extends>QToolButton</extends>
   <header>closebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
